stable/aws-events-exporter/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: aws-events-exporter
+description: "This helm chart exports aws events to prometheus via aws SQS service"
```

Comment:
If I understand correctly, it's specifically for RDS events? Like there's no general AWS events service?

If so, I think you either need to rename the chart to `aws-rds-events-exporter` or at least make it very clear in the description and/ore readme that it's for RDS events 🙂

Reply:
Phil suggested we make it generic since we fetch events from sqs queue. Therefore we can be able to use it with other services as well if we need to.

Reply:
I didn't know there was other events like that but OK, sounds like a good approach!